guess_number/guess.py

```diff
@@ -0,0 +1,71 @@
+import random
+
+random_number = random.randint(0, 100)
+
+
+def guess_number():
+
+    while True:
+        user_guess = int(input("Guess a number: "))
+        if user_guess == random_number:
+            print("Just right!")
+            return random_number
+            break
```

Comment:
There should be no need for the `break` because the `return` statement on the line above should exit the function.